nlp/2021_06_29_long_range_transformers/README.md

```diff
@@ -0,0 +1,13 @@
+# Long range transformers
+
+In recent research there has been an extensive study for improving the calculation of attention in transformer architectures. Mostly for improving their capacity to handle longer token sequences. 👊
+
+The attention calculation is known to be quadratic in compatation time with respect to the sequence length 👎. These recent advances, however, are able to perform attention calculation in linear time with respect to the sequence length. This allows us to scale the transformer architecture such that it can handle input sequences beyond the usual token length of 512.
```

Comment:
typo: compatation > computation

Reply:
These recent advance > recent advances
we haven't shown work yet, so 'these' doesn't really refer to anything

Reply:
`ble to perform attention calculation in linear time`

weeeeel, not 100%, they aim to approach linear time

Reply:
`scale the transformer architecture such that it can handle` > `scale the transformer architecture so that it can handle`

Reply:
`that it can handle input sequences beyond the usual token length of 512` > `that it can more efficiently handle input sequences beyond the usual token length of 512`
The transformer doesn't inhibit lengths beyond 512, it's just not very usable

Reply:
Thanks for the feedback Thomas! I'll change it